Makefile

```diff
@@ -276,10 +276,10 @@ cni-plugins: get-cni-sources .out-stamp build-ecs-cni-plugins build-vpc-cni-plug
 
 ifeq (${BUILD_PLATFORM},aarch64)
 run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tests
-	. ./scripts/shared_env && go test -tags integration -timeout=20m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
+	. ./scripts/shared_env && go test -tags integration -timeout=30m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
```

Comment:
this seems like a big jump. why are we doing this?

Reply:
reduced the timeout from 30 to 25